packages/sit-onyx/src/styles/variables-onyx.css

```diff
@@ -0,0 +1,141 @@
+  --onyx-color-universal-red-700: #a82824;
+  --onyx-color-universal-red-800: #93221f;
+  --onyx-color-universal-red-900: #7e1d19;
+  --onyx-color-universal-red-1000: #691714;
+  --onyx-color-universal-red-1100: #400c09;
+  --onyx-color-universal-red-1200: #400c09;
+  --onyx-color-universal-orange-100: #faf6f2;
+  --onyx-color-universal-orange-200: #f8e7d8;
+  --onyx-color-universal-orange-300: #f6d1b1;
+  --onyx-color-universal-orange-400: #f4b57e;
+  --onyx-color-universal-orange-500: #f2994a;
+  --onyx-color-universal-orange-600: #d98841;
+  --onyx-color-universal-orange-700: #bf7737;
+  --onyx-color-universal-orange-800: #a6662e;
+  --onyx-color-universal-orange-900: #8c5625;
+  --onyx-color-universal-orange-1000: #73451c;
+  --onyx-color-universal-orange-1100: #593412;
+  --onyx-color-universal-orange-1200: #402309;
+  --onyx-color-universal-green-100: #ecf8f2;
+  --onyx-color-universal-green-200: #c6e4d5;
+  --onyx-color-universal-green-300: #98d1b3;
+  --onyx-color-universal-green-400: #6ebe94;
+  --onyx-color-universal-green-500: #44aa75;
+  --onyx-color-universal-green-600: #3b9b69;
+  --onyx-color-universal-green-700: #328c5e;
+  --onyx-color-universal-green-800: #297d52;
+  --onyx-color-universal-green-900: #216d46;
+  --onyx-color-universal-green-1000: #185e3a;
+  --onyx-color-universal-green-1100: #064023;
+  --onyx-color-universal-green-1200: #064023;
+  --onyx-color-universal-purple-100: #f4f1f8;
+  --onyx-color-universal-purple-200: #dbcfea;
+  --onyx-color-universal-purple-300: #c2addc;
+  --onyx-color-universal-purple-400: #a98ace;
+  --onyx-color-universal-purple-500: #9068c0;
+  --onyx-color-universal-purple-600: #805aae;
+  --onyx-color-universal-purple-700: #704c9b;
+  --onyx-color-universal-purple-800: #603e89;
+  --onyx-number-spacing-50: 0;
+  --onyx-color-universal-purple-900: #513077;
+  --onyx-color-universal-purple-1000: #412265;
+  --onyx-number-spacing-100: 0.125rem;
+  --onyx-color-universal-purple-1100: #311452;
+  --onyx-color-universal-purple-1200: #210640;
+  --onyx-number-spacing-200: 0.25rem;
+  --onyx-number-spacing-300: 0.5rem;
+  --onyx-number-spacing-400: 1rem;
+  --onyx-number-spacing-500: 1.5rem;
+  --onyx-number-spacing-600: 2rem;
+  --onyx-number-spacing-700: 3rem;
+  --onyx-number-spacing-800: 4rem;
+  --onyx-number-spacing-900: 6rem;
+  --onyx-number-size-50: 0.0625rem;
+  --onyx-number-size-100: 0.125rem;
+  --onyx-number-size-150: 0.25rem;
+  --onyx-number-size-200: 0.5rem;
+  --onyx-number-size-250: 0.625rem;
+  --onyx-number-size-300: 0.75rem;
+  --onyx-number-size-350: 1rem;
+  --onyx-number-size-400: 1.25rem;
+  --onyx-number-size-450: 1.5rem;
+  --onyx-number-size-500: 2rem;
+  --onyx-number-size-550: 2.5rem;
+  --onyx-number-size-600: 3rem;
+  --onyx-number-size-650: 3.375rem;
+  --onyx-number-size-700: 3.625rem;
+  --onyx-number-size-750: 4rem;
+  --onyx-number-size-800: 4.5rem;
+  --onyx-number-size-850: 5rem;
+  --onyx-number-size-900: 6rem;
+  --onyx-number-size-950: 8rem;
```

Comment:
@jannick-ux As discussed, can we remove the size variables? Or if they are needed only in Figma, hide them from publishing?